From d5df9de1a618417c2fbe141f535b196c732072f8 Mon Sep 17 00:00:00 2001
From: Valentin Lab <valentin.lab@kalysto.org>
Date: Fri, 14 Jun 2013 17:58:14 +0200
Subject: [PATCH 3/4] [FIX] syntax error in default calendar view creation.

---
 openerp/osv/orm.py |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/openerp/osv/orm.py b/openerp/osv/orm.py
index 9baf58b..ee9734c 100644
--- a/openerp/osv/orm.py
+++ b/openerp/osv/orm.py
@@ -2009,7 +2009,7 @@ class BaseModel(object):
             return False
 
         view = etree.Element('calendar', string=self._description)
-        etree.SubElement(view, 'field', self._rec_name_fallback(cr, user, context))
+        etree.SubElement(view, 'field', name=self._rec_name_fallback(cr, user, context))
 
         if self._date_name not in self._columns:
             date_found = False
-- 
1.7.9.5

